dev/README.md

```diff
@@ -427,3 +427,69 @@ backwards compatible
 
 See [django-migration-linter checklist](https://github.com/3YOURMIND/django-migration-linter/blob/main/docs/incompatibilities.md)
 for the common mistakes and best practices
+
+### Removing a nullable field from a model
+
+> This only works for nullable fields (fields with `null=True` in the field definition).
+>
+> DO NOT USE THIS APPROACH FOR NON-NULLABLE FIELDS, IT WILL BREAK THINGS!
+
+1. Remove all usages of the field you want to remove. Make sure the field is not used anywhere, including filtering,
+querying, or explicit field referencing from views, models, forms, serializers, etc.
+2. Remove the field from the model definition.
+3. Generate migrations using `makemigrations` management command.
+Django should create a new migration with a `RemoveField` operation in it.
+4. Make it so that the generated migration does not actually remove the field from the database, but makes Django think
+that it has been removed. To do this, manually edit the generated migration file to use `SeparateDatabaseAndState`:
+
+    ```python
+    operations = [
+        migrations.SeparateDatabaseAndState(
+            state_operations=[
+                # Put the RemoveField operation generated by Django here
+            ],
+            database_operations=[],
+        ),
+    ]
```

Comment:
Do we need to ignore the migration linter here?

Reply:
Don't need to, check out [this migration](https://github.com/grafana/oncall/blob/ad0ed6714ac9993582e78b50d855aaf69df17619/engine/apps/alerts/migrations/0027_remove_alertreceivechannel_restricted_at_from_state.py#L10-L20)

Reply:
so it means it is reversible?

Reply:
Yes, the first migration of the two is reversible